src/main/java/com/worksap/nlp/sudachi/OovProviderPlugin.java

```diff
@@ -103,18 +103,26 @@ protected LatticeNodeImpl createNode() {
         return node;
     }
 
+    /**
+     * Recommended name for user POS flag
+     */
+    public static final String USER_POS = "userPos";
```

Comment:
In `oovPOS` used in `SimpleOovProviderPlugin`, `POS` is all uppercase, so use `userPOS` accordingly. (You can change `oovPOS` to `oovPos` if you want.)

Reply:
I don't want to break current configs, so let's make everything `userPOS` then

Reply:
Thank you!